server/routerlicious/packages/services-shared/src/restLessServer.ts

```diff
@@ -45,12 +46,16 @@ export class RestLessServer {
 
     private translateRequestFields(request: IncomingMessageEx, fields: Record<string, any>): void {
         // Parse and override HTTP Method
-        const methodOverride = fields[
+        const methodOverrideField = fields[
             RestLessFieldNames.Method
-        ] as string;
-        request.method = methodOverride;
+        ] as string[] | string;
```

Comment:
Curious, was changing the type of variables produced by reading `fields` to `string[] | string` something related to formidable/inclusion?

Reply:
Yes, Formidable now always returns `fields[*]` as an array as of v3.0.0